app/javascript/retrospring/features/answerbox/index.ts

```diff
@@ -0,0 +1,20 @@
+import registerEvents from "retrospring/utilities/registerEvents";
+import {createShareEvent} from "./share";
+
+export default (): void => {
+  if (!('share' in navigator)) {
+    return;
+  }
```

Comment:
Good, but this also breaks setting any `answerbox` related functionality if `navigator.share` is not there, maybe move this check to `createShareEvent`.